include/pybind11/pybind11.h

```diff
@@ -80,6 +80,8 @@ class cpp_function : public function {
     }
 
     /// Construct a cpp_function from a class method (non-const, lvalue ref-qualifier)
+    /// Is a copy of the overload for non-const functions without explicit ref-qualifier
```

Comment:
Maybe delete "Is" here and below?
// A copy of ...

Reply:
... and done :+1: